.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
```

Comment:
in case you are wondering this image is built from -- https://github.com/freedomofpress/docker-debian-systemd . Probably want to rebrand and shove under the fpf banner (instead of under the docker hub `msheiny` name) in the future.